composer.json

```diff
@@ -34,14 +34,28 @@
     "fzaninotto/faker": "~1.9.0",
     "friendsofphp/php-cs-fixer": "2.*",
     "squizlabs/php_codesniffer": "3.*",
-    "php-coveralls/php-coveralls": "~2.2.0"
+    "php-coveralls/php-coveralls": "~2.2.0",
+    "brainmaestro/composer-git-hooks": "^2.8"
   },
   "autoload": {
     "psr-4": {
       "BigBlueButton\\": "src"
     }
   },
   "scripts": {
-    "test": "vendor/bin/phpunit"
+    "test": "vendor/bin/phpunit",
+    "post-install-cmd": "vendor/bin/cghooks add --ignore-lock",
+    "post-update-cmd": "vendor/bin/cghooks update"
+  },
+  "extra": {
+    "hooks": {
+      "pre-commit": [
+        "vendor/bin/php-cs-fixer fix --dry-run --allow-risky=yes"
```

Comment:
Why is it --dry-run?

Reply:
Because I only want to test if there are errors and not change anything.

Reply:
Ok, are there plans to apply the coding style automatically or is that the responsibility of the developer? What exactly is our coding style?

Reply:
To apply cs-fixer automatically, we would need to stash all changes before and amend the changes afterwards. There is way to many things which could go wrong in my opinion.

> What exactly is our coding style?

We basically use the same as upstream. See the `.php_cs` file.

Reply:
Thanks